python/kwiver/vital/modules/module_helpers.cxx

```diff
@@ -114,27 +76,31 @@ register_factories(kwiver::vital::plugin_loader& vpm)
     // Release the GIL
     PyEval_SaveThread();
   }
+}
 
-  _load_python_library_symbols();
-
+/*
+ * check if the python shared library symbols have been loaded
+ * From: https://stackoverflow.com/questions/50570223/dlopenrtld-noload-still-returns-not-null-after-dlclose
+ */
+bool is_python_library_loaded(const std::string& python_library_path)
+{
+  void *handle = dlopen(python_library_path.c_str(), RTLD_NOW | RTLD_NOLOAD);
```

Comment:
I mentioned it over email but the call to dlopen will fail when Linux header dlfcn.h isn't there, e.g. Windows. You should use kwiversys' DynamicLoader instead.